src/screens/FeedScreen/index.js

```diff
@@ -224,12 +223,6 @@ const FeedScreen = (props) => {
     />
   );
 
-  const onWillSendViewPostTime = (event) => {
-    if (isSamePostViewed(event)) return;
-    sendViewPostTime();
-    updateViewPostTime(event);
-  };
-
   const renderItem = ({item, index}) => {
     if (item.dummy) return <React.Fragment key={index} />;
     if (updateMoreText && updateIndex === index) return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onWillSendViewPostTime` function has been removed and replaced with a call to `useViewPostTimeHook`. Ensure that the new hook provides the same functionality as the old function.



<!-- This is an auto-generated comment by OSS CodeRabbit -->